ignite3/src/main/java/site/ycsb/db/ignite3/AbstractSqlClient.java

```diff
@@ -1,32 +1,106 @@
 package site.ycsb.db.ignite3;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 import site.ycsb.ByteIterator;
+import site.ycsb.DBException;
 
 abstract class AbstractSqlClient extends IgniteAbstractClient {
+  /** Prepared statement for reading values. */
+  protected static final ThreadLocal<PreparedStatement> READ_PREPARED_STATEMENT = new ThreadLocal<>();
 
-  static String prepareInsertStatement(String table, String key, Map<String, ByteIterator> values) {
-    List<String> columns = new ArrayList<>(Collections.singletonList(PRIMARY_COLUMN_NAME));
-    List<String> insertValues = new ArrayList<>(Collections.singletonList(key));
+  /** Prepared statement for inserting values. */
+  protected static final ThreadLocal<PreparedStatement> INSERT_PREPARED_STATEMENT = new ThreadLocal<>();
 
-    for (Map.Entry<String, ByteIterator> entry : values.entrySet()) {
-      columns.add(entry.getKey());
-      insertValues.add(entry.getValue().toString());
+  /** SQL string of prepared statement for reading values. */
+  protected static String readPreparedStatementString;
+
+  /** SQL string of prepared statement for inserting values. */
+  protected static String insertPreparedStatementString;
```

Comment:
This field is static
```suggestion
  protected static String INSERT_PREPARED_STATEMENT_STRING;
```

Reply:
It's against the check style rules. Only `static final` must be in the uppercase.